src/Package/Impl/Options/R/RToolsOptionsPage.cs

```diff
@@ -232,7 +235,10 @@ class SettingsHolder {
                 _dict[name] = value;
             }
 
-            public void Apply() => _settings.SetProperties(_dict);
+            public void Apply() {
+                _settings.SetProperties(_dict);
+                ((IRPersistentSettings)_settings).SaveSettingsAsync().DoNotWait();
```

Comment:
I guess... Point was to limit widely exposed functionality, sort of like IWpfTextView/ITextView. Maybe do it the other way around: derive IRPersistentSettings from IRToolsSettings and export it.